tutorials/setting-up-redis.md

```diff
@@ -0,0 +1,250 @@
+---
+title: How to Set Up Redis on Google Compute Engine
+description: Learn how to get Redis running on Google Compute Engine.
+author: chingor13
+tags: Compute Engine, Redis
+date_published: 2017-06-08
+---
+This tutorial shows how to set up [Redis](https://redis.io/) on
+Google Cloud Platform in just a few minutes. Follow this tutorial to configure a standalone
+Redis instance on a Debian 8 (jessie) virtual machine instance on Compute Engine.
+
+You can also use [Cloud Launcher](https://console.cloud.google.com/launcher/details/click-to-deploy-images/redis)
+to set up Redis cluster on Compute Engine with just a few clicks.
+
+
+## Objectives
+
+* Install Redis on a Compute Engine instance.
+* Configure Redis for remote access.
+* Configure a Cloud Platform firewall to open a port. (optional)
+* Connect to Redis from a remote computer. (optional)
+
+## Before you begin
+
+You'll need a Google Cloud Console project. You can use an existing project or
+click the button to create a new project.
+
+**[Create a project](https://console.cloud.google.com/project)**
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform,
+including:
+
++ Google Compute Engine
+
+Use the [pricing calculator](https://cloud.google.com/products/calculator/#id=88a62458-4ab4-42fa-a7f3-e6b1d66bd307)
+to generate a cost estimate based on your projected usage. New Cloud Platform users might be
+eligible for a [free trial](https://cloud.google.com/free-trial).
+
+
+## Creating a Compute Engine instance
+
+For the purposes of this tutorial, the default machine type works fine, so
+you don't need to change the default setting.
+In production, you need to decide how much computing power is required
+for your application. In general, database systems tend to be more
+constrained by I/O bottlenecks and hard disk speed than by CPU capabilities.
```

Comment:
Sure, we can add a short note about memory. We don't want to get into too many details about configuration and tuning as it's beyond the scope of this tutorial.